kura/org.eclipse.kura.driver.ble.xdk/OSGI-INF/metatype/org.eclipse.kura.driver.ble.xdk.xml

```diff
@@ -29,7 +29,7 @@
             cardinality="0"
             required="true"
             default="flase"
-            description="Enable Rotation-Quaternion or Disable the Accelerometer and Gyroscope"/>
+            description="Set true enable Rotation-Quaternion otherwise set flase enalbe the Accelerometer and Gyroscope"/>
```

Comment:
Maybe: "Set this field as true to enable quaternion representation. When this field is false, the output of accelerometer, gyroscope and magnetometer is returned."